benchmarks/benchmarks/__init__.py

```diff
@@ -1 +1,48 @@
 from . import common
+import sys, os
+
+def show_cpu_features():
+    from numpy.lib.utils import _opt_info
+    info = _opt_info()
+    info = "NumPy CPU features: " + (info if info else 'nothing enabled')
+    # ASV wrapping stdout & stderr, so we assume having a tty here
+    if 'SHELL' in os.environ and sys.platform != 'win32':
+        # to avoid the red color that imposed by ASV
+        print(f"\033[33m{info}\033[0m")
+    else:
+        print(info)
+
+def dirty_lock(lock_name, lock_on_count=1):
+    # this lock occurred before each round to avoid duplicate printing
+    if not hasattr(os, "getppid"):
+        return False
+    ppid = os.getppid()
+    if not ppid or ppid == os.getpid():
+        # not sure if this gonna happen, but ASV run each round in a separate process
+        # so the lock should be based on the parent process id only
+        return False
+    lock_path = os.path.join(os.environ.get("ASV_ENV_DIR", ''), "project", lock_name)
```

Comment:
```suggestion
os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "env", lock_name))
```
 avoid env var  `ASV_ENV_DIR`, because ASV swaps the project dir and clears the lock file.

Reply:
done